ci/matrix.xml

```diff
@@ -225,6 +228,8 @@
         javalibExTestSuite/test:scalastyle helloworld/scalastyle \
         reversi/scalastyle testingExample/scalastyle \
         testingExample/test:scalastyle
+    sbt ++$scala ir/compile:doc tools/compile:doc jsEnvs/compile:doc \
```

Comment:
Oh. That was probably a mistake. Yes, please add `&&` between consecutive invocations of sbt.